### ISSUE TITLE
JGRAPHX IS NOT WORKING IN NETBEANS

### ISSUE THREAD

**Issue by gentlemanjos1** [jgraph/jgraphx#62]
Hello Everyone:

I am developing a java project using Netbeans and the jgraphx library, I first tested it in eclipse and it went well, but when I tried to adapt it and run it in Netbeans, It doesn´t display any of the mxGraph components in Jpanel or in JFrame. I am testing a function I implemented myself for adding nodes, but the frame doesn´t show anything.

I have already imported the .jar file, I have searched and followed other tutorials, but nothing works.

here are some screenshots that explain better the problem:

WORKING WITH ECLIPSE
![image](https://cloud.githubusercontent.com/assets/22244863/18611122/4e1acad0-7cf6-11e6-99c1-c1d8abc3de5f.png)

![image](https://cloud.githubusercontent.com/assets/22244863/18611124/599d5706-7cf6-11e6-9c2e-892e206af869.png)

![image](https://cloud.githubusercontent.com/assets/22244863/18611125/60ba9882-7cf6-11e6-8c87-015ae0dd0347.png)

As you will see, it works well in Eclipse.

WORKING WITH NETBEANS
![image](https://cloud.githubusercontent.com/assets/22244863/18611128/6cb4ae3e-7cf6-11e6-95e4-990eaa0efc52.png)

![image](https://cloud.githubusercontent.com/assets/22244863/18611130/746a1754-7cf6-11e6-88f7-1d7aafa02619.png)

![image](https://cloud.githubusercontent.com/assets/22244863/18611131/7bbce392-7cf6-11e6-8629-df86bf1421ef.png)

As you will see, in Netbeans GUI doesn´t display anything

So, I request your help with htis issue so I can continue with my project. I send the code (both eclipse and netbeans) in attached .pdf files (This messaging tool didn´t allow .zip format), so you please can examine it and tell me what the problem is.

Thank you very much for your attention and sooner answer

[Eclipse code version.pdf](https://github.com/jgraph/jgraphx/files/478413/Eclipse.code.version.pdf)
[Netbeans code version.pdf](https://github.com/jgraph/jgraphx/files/478412/Netbeans.code.version.pdf)

**Comment by gentlemanjos1** [comment]
Hello,

I am the github user gentlemanjos1
I have posted an issue titled JGRAPHX IS NOT WORKING IN NETBEANS(#61 and #62) twice, but it has been closed two times and I don´t understand why; I recongnize that in my first post I authored it with my name, then I removed my name and posted a new issue, but it was closed again.
 I would appreciate if you guide me hoy to post my issue properly so I can get the help I need.
Thank you very much for your attention and sooner answer.

**Comment by davidjgraph** [comment]
Support for how to use an IDE is outside the scope of supporting this library. As you proved with Eclipse it works correctly when used correctly. Please contact the Netbeans team for support using the tool.

**Comment by gentlemanjos1** [comment]
Hello:
I am the Github user gentlemanjos1
I contacted netbeans support team and I got no answer.
I understand you are not responsible for Netbeans IDE performance, but JGRAPHX performance in an IDE is of your concern; all I need to know is how to make it work in Netbeans (which is also a java development platform), or, Does the library work only in eclipse?
Thanks for your attention and sooner answer.
